unity-renderer/Assets/DCLServices/MapRendererV2/MapRenderer.cs

```diff
@@ -98,13 +99,35 @@ private void ReleaseCamera(IMapCameraControllerInternal mapCameraController)
             mapCameraPool.Release(mapCameraController);
         }
 
+        private bool isSatelliteViewMode = true;
+
+        public void SetSatelliteViewMode(bool isActive)
```

Comment:
You should not manipulate with layers directly: there is already logic for this in `EnableLayers` and `DisableLayers`: `ActivityOwners` means that there could be several consumptions of the map: e.g. `MiniMap` and `Map`, it should be respected.
In order to implement it properly you should:
Slightly modify the aforementioned methods to: 
 - allow disable/enable layers incrementally from the active customer dynamically.
 - expose them
 - Make `ActivityOwners` respect reconfiguration of layers from the same consumer (currently it supposes methods are called only once on rental and on disposal respectively)

Reply:
@davidejensen you should have a similar implementation with layers' capability being activated/deactivated